src/mod/asr_tts/mod_unimrcp/mod_unimrcp.c

```diff
@@ -3890,11 +3893,13 @@ static apt_bool_t recog_stream_read(mpf_audio_stream_t *stream, mpf_frame_t *fra
 	}
 
 	/* grab the data.  pad it if there isn't enough */
-	if (speech_channel_read(schannel, frame->codec_frame.buffer, &to_read, 0) == SWITCH_STATUS_SUCCESS) {
+	if ((status = speech_channel_read(schannel, frame->codec_frame.buffer, &to_read, 0)) == SWITCH_STATUS_SUCCESS) {
```

Comment:
So instead of doing this in a single line, separate it into two line code to make it readable and clean.
```
status = speech_channel_read(schannel, frame->codec_frame.buffer, &to_read, 0);
if (status == SWITCH_STATUS_SUCCESS) {
 ....
} else if (status == SWITCH_STATUS_FALSE) {
 ....
}
```